### ISSUE TITLE
How can we merge efforts?

### ISSUE THREAD

**Issue by OR13** [w3c/note-respec-repo-template#2]
See https://github.com/w3c/respec/issues/3122

Happy to open a PR to merge any / all of this if it's valuable here.

Please review and let me know how I can help.

**Comment by plehegar** [comment]
This repository is more than a simple respec template. It also contains templates for GH labels, settings, etc. The intention is to revise our [IP repository manager](https://github.com/w3c/ash-nazg) to use this template for the case of a W3C Group Note. So, given the specificiy, I'm not sure if we can/shouod merge the efforts.

**Comment by OR13** [comment]
@plehegar agree, sounds like I was directed here by someone who thought these things were more similar than they are, you are welcome to close this issue.

If you can think about anyone / anywhere my "respec github pages repo template"  might better live, feel free to forward it / link me along.
